src/audio/mux/mux.c

```diff
@@ -44,12 +44,51 @@ DECLARE_SOF_RT_UUID("demux", demux_uuid, 0xc4b26868, 0x1430, 0x470e,
 
 DECLARE_TR_CTX(demux_tr, SOF_UUID(demux_uuid), LOG_LEVEL_INFO);
 
+static bool mux_mix_check(struct sof_mux_config *cfg)
```

Comment:
Can we have a comment here describing at a high level what we are checking.

Reply:
added something...